infrastructure/src/main/java/org/corfudb/infrastructure/CorfuMsgHandler.java

```diff
@@ -36,7 +38,7 @@
 @Slf4j
 public class CorfuMsgHandler {
 
-    private static final String CORFU_INFRASTRUCTURE_PREFIX = "corfu.infrastructure.message-handler.";
+    private final Map<CorfuMsgType, String> timerNameCache = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this Map to an EnumMap. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1640)